sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -226,14 +230,31 @@ def _management_request(self, mgmt_msg, op_type):
                     mgmt_msg,
                     constants.READ_OPERATION,
                     op_type=op_type,
-                    status_code_field=b"status-code",
-                    description_fields=b"status-description",
+                    status_code_field=MGMT_STATUS_CODE,
+                    description_fields=MGMT_STATUS_DESC,
                 )
-                status_code = response.application_properties[b"status-code"]
+                status_code = response.application_properties[MGMT_STATUS_CODE]
                 if status_code < 400:
                     return response
-                raise errors.AuthenticationException(
-                    "Management request error. Status code: {}".format(status_code)
+                if status_code in [401]:
```

Comment:
elif

Reply:
@YijunXieMS , if and elif either applies here as the code in the statement either raise or return - no more code would be executed.

Reply:
It should be `if` so long as the previous block raises or returns.